Tools/px4airframes/rcout.py

```diff
@@ -50,14 +50,23 @@ def __init__(self, groups, board):
                 result +=   "# %s\n" % param.GetName()
                 result +=   "if param compare SYS_AUTOSTART %s\n" % id_val
                 result +=   "then\n"
-                result +=   "\tsh %s\n" % path
+                result +=   "\tsh /etc/init.d/airframes/%s\n" % path
+                result +=   "\tset SYS_AUTOSTART_VALUE %s\n" % id_val
```

Comment:
This increases flash usage (that is why there was a `cd`).
Can you find a solution that does not add to to each airframe?
For example:
```
result +=   "\set F %s\n" % path
```
And then you can check if `F` is valid at the end.

Also we'll need to coordinate with #12526.

Reply:
@bkueng 
Thanks for the feedback!
I changed it so to reduce the flash usage from my previous commit.
also,  coordinated it with #12526